doc/api/process.md

```diff
@@ -1247,6 +1247,22 @@ process.kill(process.pid, 'SIGHUP');
 *Note*: When `SIGUSR1` is received by a Node.js process, Node.js will start
 the debugger, see [Signal Events][].
 
+## process.libc
+<!-- YAML
+added:
+-->
+
+* {string}
+
+The `process.libc` property returns the C standard library used to compile
+the Node.js binary. For instance `glibc` or `musl`.
+
+<!-- eslint-disable semi -->
```

Comment:
A nit: why is this disabling with the semicolon used?

Reply:
I copied that part from somewhere else, I didn't take a close look at the comment I must admit. I will remove.